explainaboard/processors/conditional_generation.py

```diff
@@ -156,31 +145,54 @@ def default_analyses(self) -> list[AnalysisLevel]:
                 ),
             ),
         }
-        tok_continuous_features = [
-            k for k, v in tok_features.items() if ('float' in unwrap(v.dtype))
-        ]
-        tok_analyses: list[BucketAnalysis] = [
-            BucketAnalysis(
-                description=tok_features[x].description, feature=x, method="continuous"
-            )
-            for x in tok_continuous_features
-        ]
 
         return [
             AnalysisLevel(
                 name='example',
                 features=examp_features,
                 metric_configs=self.default_metrics(level='example'),
-                analyses=cast(List[Analysis], examp_analyses),
             ),
             AnalysisLevel(
-                name='tok',
+                name='token',
```

Comment:
This looks a breaking change. Be careful to check if this change won't affect anything.

Reply:
This PR is modifying the v0.11 branch, which isn't yet merged into the main branch, so I think it's OK.
If this were already merged into the main branch it would indeed break backward compatibility with previously generated reports so we'd need to be a bit more careful.

Reply:
Oh okay, but could you avoid merging to any non-main branches? Merging to arbitrary branches makes us hard to track the actual progress of the development.

Reply:
That makes sense. The justification behind doing it here is because I wanted to make it easier for you to review these changes independently of the very large PR that we're currently waiting to merge here (once you approve of it of course): https://github.com/neulab/ExplainaBoard/pull/372

Reply:
Ah sure. It's somewhat unfortunate since GitHub lacks feature to compare different base commit than main.